Exercises/loops/loops.js

```diff
@@ -16,3 +25,24 @@ const arrayToFilter = Array.from(new Array(1000), () => {
 
   return null;
 });
+console.log(arrayToFilter);
+
+//2. Filter array `arrayToFilter` from non numeric values and `NaN`.
+const results = arrayToFilter.filter(value => typeof value === 'number' && !isNaN(value));
+console.log(results);
+
+//3. Make while loop until sum of randomly generated values will be greater than `200` and save amount of iterations.
+function sum(array) {
```

Comment:
Values thould be generated randomly on every loop iteration until it's sum will be greater than 200